server/fakeDate.js

```diff
@@ -0,0 +1,29 @@
+export const user1 = {
```

Comment:
how are you inserting these fake users into GraphCool?

Reply:
this is just a fake data object for people working on the front end before all the queries are written for each user field.